crates/runtime/benches/bench.rs

```diff
@@ -179,19 +182,24 @@ async fn bench_main() -> Result<(), String> {
                 _ => return Err(format!("Invalid mode parameter for {accelerator} accelerator")),
             };
 
-            let acceleration = create_acceleration(accelerator, mode, args.bench_name.as_ref());
+            let append_mode = args.append_mode.unwrap_or(false);
```

Comment:
It would be clearer if this was an enum, instead of a bool. Then in the match statement it can match on `Mode::Append, "tpch"` which is much clearer